### DISCUSSION TITLE
Welcome to the Intro to Git workshop!

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/git-rstudio-basics#10] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with community members who participated in our workshops! 

We hope that you ask the questions you’re wondering about after this workshop here! To do this [select the Q&A category](https://github.com/epiverse-trace/git-rstudio-basics/discussions/new/choose). The instructors and the RSE developers at the [Epiverse-TRACE](https://epiverse-trace.github.io/) we'll be able to provide you support!

(If you wish, take a look and feel free to [participate in our ongoing discussions](https://github.com/orgs/epiverse-trace/discussions)! 🥊💢)

To get started, Share with us a link to the `case` repository you just uploaded! (following the steps 🧟‍♀️  of Dracula 🧛 and Wolfman 🐺 )

**Comment by Beth96** [comment]
Thanks for the really useful training course! 
URL:  https://github.com/Beth96/cases.git

**Reply by supingling** [reply]
A really good introduction. URL:https://github.com/supingling/cases

**Comment by ruoranepi** [comment]
Thank you! I particularly appreciate the last 5 min of self-reflection time. https://github.com/ruoranepi/gitdemo.git

**Reply by Beth96** [reply]
Same!!

**Comment by leawiedmann** [comment]
thank you for the course! https://github.com/leawiedmann/cases.git

**Comment by hiratanvir** [comment]
This is how we resolved the error Ayaka was getting when git pushing on to remote branch:
https://stackoverflow.com/questions/43378060/meaning-of-the-github-message-push-declined-due-to-email-privacy-restrictions

**Comment by rayensun8** [comment]
Hola! va mi repositorio https://github.com/rayensun8/reproducibilidad-de-datos

**Comment by aguayovalentina** [comment]
Gracias por el taller! Nos vemos este jueves para la siguiente parte.
https://github.com/aguayovalentina/GitHub-comienzo

**Comment by TomasBralic** [comment]
Muchas gracias por la capacitación!
https://github.com/TomasBralic/casos.git

**Comment by PValenzuela-Carvajal** [comment]
Hola, aquí va mi repositorio:) https://github.com/Lasfloresecaen/Clase

**Comment by CamCastillo** [comment]
Hola! Este es mi repositorio https://github.com/CamCastillo/Casos  gracias por el taller!

**Comment by esdrasjyl** [comment]
hola, gracias por el taller, todo claro y facil de aprender.

https://github.com/esdrasjyl/casos

**Comment by CarolinaBlanco** [comment]
Gracias ! Comparto mi repositorio :)

https://github.com/CarolinaBlanco/casos

**Comment by ignaciocastroa** [comment]
Hola! Muchas gracias por la brillante explicación. Les dejo mi repositorio. 
https://github.com/ignaciocastroa/casos.git

**Comment by Vaiti13** [comment]
Hola! muchas gracias por el taller,  aprendí mucho, aquí va mi enlace de lo que trabajado en el taller de hoy: https://github.com/Vaiti13/casos

**Comment by ghost** [comment]
Hola, muy buen taller! muchas gracias.

https://github.com/ricardoahumada-ssms/casos

**Comment by Rincus** [comment]
Gracias por el taller va mi repositorio: https://github.com/ignaciocastroa/casos.git

**Comment by SCCadiz** [comment]
Gracias por el taller, buenísimo [ https://github.com/SCCadiz/casos](url)

**Comment by EstrategiasDigitalesSSMS** [comment]
¡Hola! Acá Juan Cisternas. Muy agradecido del taller. Muy conforme. Va mi repositorio:

https://github.com/EstrategiasDigitalesSSMS/casos-taller-.git

**Comment by nvergaramallegas** [comment]
https://github.com/nvergaramallegas/casos.git muchas gracias y nos vmos el jueves

**Comment by FabiAngeles** [comment]
Thank you for the event! I hope I use the tool.

**Comment by gigicnj** [comment]
aprendí un montón  https://github.com/claudianj/casos :)

**Comment by VicenteRobinson** [comment]
was a truly noteworthy experience. Here is my repo: https://github.com/[VicenteRobinson/casos](https://github.com/VicenteRobinson/casos)

**Comment by javiersilvaalvarado** [comment]
muchas gracias https://github.com/[javiersilvaalvarado/casos](https://github.com/javiersilvaalvarado/casos)

**Comment by tmalfaro42** [comment]
Muchas gracias por el taller.
Mi repositorio
https://github.com/tmalfaro42/Casos
